src/Makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY: all
+all: help
+
+.PHONY: lint
+lint: check-isort check-black ## Run all linters and formatters
+
+.PHONY: black
+black: ## Check Python source code formatting with black
+	@black .
+
+.PHONY: check-black
+check-black: ## Update Python source code formatting with black
+	@black --check --diff .
+
+.PHONY: isort
+isort: ## Check Python import organization with isort
+	@isort .
+
+.PHONY: check-isort
+check-isort: ## Update Python import organization with isort
+	@isort --check-only --diff .
+
+# Explanation of the below shell command should it ever break.
+# 1. Set the field separator to ": ##" and any make targets that might appear between : and ##
+# 2. Use sed-like syntax to remove the make targets
+# 3. Format the split fields into $$1) the target name (in blue) and $$2) the target descrption
+# 4. Pass this file as an arg to awk
+# 5. Sort it alphabetically
+# 6. Format columns with colon as delimiter.
+.PHONY: help
+help: ## Print this message and exit.
+	@printf "Makefile for developing and testing the SecureDrop proxy.\n"
```

Comment:
Yes, I copied this out of the securedrop-proxy repo. Can we come up with some text here that's generic without needing to specify the project/repo name?

Reply:
We could also use a bit of `Makefile` trickery. At the very top, assign the output of a command to a variable, like `securedrop-updater` sort of does in `scripts/common.sh` and using the git origin url to derive the project's name. That variable could then just be used there.

Reply:
Yeah, that would work. Taking a step back, if we dropped this line, would we lose any value? I don't really think so...

Reply:
I agree.  One advantage of using Make as our command-runner here is that it's tied strictly to your working directory.  I don't think we need to remind people what directory they're in, which is effectively what this `printf` does.  :-)